doc/index.md

```diff
@@ -0,0 +1,338 @@
+    }
+
+    // Set viewpoint of the map depending on user's setting
+    await this.SetInitialViewPointAsync().ConfigureAwait(false);
+}
+```
+
+The Initial Viewpoint is the initial extent that the map is loaded to when the application starts. This extent is configured in the settings. If the initial extent is not set, the map will display the full extent of the mmpk.
+
+### Navigating the map
+
+There are three ways to move around the map: pan, zoom and floor change. The floor picker is displayed only when the map is zoomed in past a certain extent. This extent is also defined in the settings and can be easily changed. When the floor picker is activated, the app applies [definition expressions](/net/latest/ios/api-reference/) on the rooms and walls layers to only display the selected floor.
+
+![Manual Zoom-in](./images/manualzoomin.png)
+
+```csharp
+for (int i = 1; i < Map.OperationalLayers.Count; i++)
+{
+    ...
+    // select chosen floor
+    featureLayer.DefinitionExpression = string.Format(
+                        "{0} = '{1}'",
+                        AppSettings.CurrentSettings.RoomsLayerFloorColumnName,
+                    this.SelectedFloorLevel);
+    ...
+}
+```
+
+### Search functionality
+
+The mmpk locator is used to search for offices or employees. When user starts entering text, the locator is used to provide suggestions.
+
+|Auto-complete office|Auto-complete name|
+|---	|---	|
+|![Auto-complete office](./images/autocompleteoffice.png)|![Auto-complete name](./images/autocompletename.png)|
+
+```csharp
+internal async Task<IReadOnlyList<SuggestResult>> GetLocationSuggestionsAsync(string userInput)
+{
+    var locatorInfo = this.Locator.LocatorInfo;
+
+    if (locatorInfo.SupportsSuggestions)
+    {
+        // restrict the search to return no more than 10 suggestions
+        var suggestParams = new SuggestParameters { MaxResults = 10 };
+
+        // get suggestions for the text provided by the user
+        var suggestions = await this.Locator.SuggestAsync(userInput, suggestParams);
+        return suggestions;
+    }
+    return null;
+}
+```
+
+When a suggestion is selected, the selection is then passed to the locator again, this time to retrieve the actual location
+
+```csharp
+internal async Task<GeocodeResult> GetSearchedLocationAsync(string searchString)
+{
+    // Geocode location and return the best match from the list
+    var matches = await this.Locator.GeocodeAsync(searchString);
+    var bestMatch = matches.FirstOrDefault();
+    return bestMatch;
+}
+```
+
+Several matches are returned, with corresponding accuracy scores. This app makes the assumption that the first match is the best, but it could be changed so users see more than one of the returned matches.
+[Read more about locators](/net/latest/wpf/guide/search-for-places-geocoding-.htm).
+
+The resulting location is then added to the map inside a [Graphics Overlay](/net/latest/wpf/guide/add-graphics-overlays-to-your-app.htm)
```

Comment:
`[Graphics Overlay](/net/latest/wpf/guide/add-graphics-overlays-to-your-app.htm)`

Relative path needs the https://developers.arcgis.com before it